_overviews/scala3-book/scala-for-java-devs.md

```diff
@@ -25,13 +25,13 @@ It presents the similarities and differences between Java and Scala at a high le
 
 At a high level, Scala shares these similarities with Java:
 
-- Scala code is compiled to *.class* files, packaged in JAR files, and runs on the JVM
-- It’s an object-oriented programming (OOP) language
+- Scala code is compiled to _.class_ files, packaged in JAR files, and runs on the JVM
+- It’s an [object-oriented programming][modeling-oop] (OOP) language
 - It’s statically typed
-- Both languages have support for immutable collections, lambdas, and higher-order functions
+- Both languages have support for immutable collections, lambdas, and [higher-order functions][hofs]
```

Comment:
Sorry, this might be a stupid question, but which immutable collections in Java are you referring to? As far as I remember, most collections in the stdlib in Java are mutable. There exists Guava, but that's not the stdlib. Maybe things improved and I just did not notice it :)

Reply:
When I looked into this I started with the `List`, `Map`, and `Set` links on [this page](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Collections.html), and found that these methods create collections that are “unmodifiable”: `List.of`, `List.copyOf`, `Map.of`, `Map.ofEntries`, `Map.copyOf`, `Set.of`, and `Set.copyOf`. They then discuss what they mean by _unmodifiable_ in the Unmodifiable Collections section on [this page](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Collection.html#unmodifiable).

Reply:
Ok, great - I happily retract :) given that these exists already for a long time I cannot recall what my problem with immutable collections was back in the time. Thanks for clarifying.

Reply:
Okay, I modified that line to only mention lambdas and HOFs, and added a line under “Differences” about the immutable collections.